### PR TITLE
Functional Tests - Add test 'Choose address' in create order page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add test 'Choose address' in create order page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/08_chooseAddress.js [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+    it('should get the order ID', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'getOrderID', baseContext);
+
+      orderID = await orderPageCustomerBlock.getOrderID(page);
+      await expect(orderID).to.be.at.least(5);
+    });
+  });
+
+  // 2 - Create second order
+  describe('Create first order and choose the created address in PRE-TEST', async () => {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/08_chooseAddress.js]:
```suggestion
  describe('Create second order and choose the created address in PRE-TEST', async () => {
```

File: tests/UI/pages/FO/myAccount/orderDetails.js [hunk 2]
```diff
@@ -110,6 +110,24 @@ class OrderDetails extends FOBasePage {
...
     await this.clickAndWaitForNavigation(page, this.reorderLink);
   }
+
+  /**
+   * Get delivery address
+   * @param page {Page} Browser tab
+   * @returns {Promise<string>}
+   */
+  async getDeliveryAddress(page) {
+    return this.getTextContent(page, '#delivery-address');
```
Comment (Progi1984) [inline tests/UI/pages/FO/myAccount/orderDetails.js]: Could you move the selector to selectors list in constructor ?

File: tests/UI/pages/FO/myAccount/orderDetails.js [hunk 3]
```diff
@@ -110,6 +110,24 @@ class OrderDetails extends FOBasePage {
...
+  }
+
+  /**
+   * Get Invoice address
+   * @param page {Page} Browser tab
+   * @returns {Promise<string>}
+   */
+  async getInvoiceAddress(page) {
+    return this.getTextContent(page, '#invoice-address');
```
Comment (Progi1984) [inline tests/UI/pages/FO/myAccount/orderDetails.js]: Could you move the selector to selectors list in constructor ?

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/08_chooseAddress.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/08_chooseAddress.js]: done thx :+1:

File: tests/UI/pages/FO/myAccount/orderDetails.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/myAccount/orderDetails.js]: done thx :+1:

File: tests/UI/pages/FO/myAccount/orderDetails.js (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/myAccount/orderDetails.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**